README.md

```
@@ -2,313 +2,1534 @@
+- [Mayank Mehra](https://mayank-portfolio-pied.vercel.app)
+- [Maytiya Monburinon](https://gigigimay.github.io)
+- [Md Kawsar](https://www.kawsarlog.com) [Python Developer | Web Scraping, Automation, And Development Expert]
+- [Md Muhaiminul Islam Shihab](https://muhaiminShihab.github.io) [Software Developer | Php, Laravel, Next.Js, Mysql.]
+- [Md Nabil Ahsan](https://www.mdnabilahsan.com)
+- [Md Nuruzzaman](https://mdnuruzzamandev.web.app) [Frontend Developer]
+- [Md Rashid](https://md-rashid.vercel.app) [Software Developer]
+- [Md Usman Ansari](https://mdusmanansari.netlify.app)
+- [Mees Verberne](https://meesverberne.com)[Creative Frontend Developer & Designer]
+- [Meet Vipul Gandhi](https://portfolio-nu-lovat-90.vercel.app/project) [Software Developer]
+- [Mehdi Ali](https://exoo25.github.io)
+- [Mehedi Hasan] (https://m-hasan.vercel.app) [Frontend Developer]
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)